internals/webpack/webpack.config.dev.babel.js

```diff
@@ -20,7 +20,6 @@ module.exports = require('./webpack.base.babel')({
 
   // Don't use hashes in dev mode for better performance
   output: {
-    filename: '[name].js',
```

Comment:
This is the default value, the migration document suggested to remove it.


https://webpack.js.org/migrate/5/#clean-up-configuration